CHANGELOG

```diff
@@ -1,3 +1,7 @@
+Version 2.2.0 (2022-11-01)
```

Comment:
Not a big deal, but should this date be adjusted as well? ;)

Reply:
Aaah yes, I will fix this. Thanks.

Reply:
Ooops, this should have been a PR against develop so I'll actually remove the Prepare for release... commit.